Gear item blocking (#359)
Lembas-Modding-Team/pvp-mode: src/main/resources/assets/pvpmode/modules/lotr/default_blocked_gear.txt

#

> # This file lists all gear items, that are blocked under certain conditions (alignment, pledging, PvP, PvE, faction).

Reviewer: ```suggestion
# This file lists all gear items that are blocked under certain conditions (alignment, pledging, PvP, PvE, faction).
```

#

> # You can assign a faction an alignment value and a list of items, players can only use these items, if they have at least the specified alignment with the specified faction.

Reviewer: ```suggestion
# You can assign a faction an alignment value and a list of items. Players can only use these items if they have at least the specified alignment with the specified faction.
```

# It's also possible to specify - in addition to the alignment criterion - that the player needs to pledge to the specified faction.
#

> # One can include multiple lines starting with the same faction, but with different alignment values. That way you can specify different conditions for different items.

Reviewer: ```suggestion
# You can include multiple lines starting with the same faction, but with different alignment values. This way you can specify different conditions for different items.
```

# Note that if you specify two entries with the same faction, but different alignment values, for the same item, the condition with the lowest alignment value applies.
#
# If a placeholder is specified, the alignment condition has to apply to at least one faction assigned to the placeholder.
#
# A configuration entry consists of one line, which contains three or four columns separated by a semicolon (;).
#

> # The first part is the faction name, which is the name which is internally used by the game (see the command /alignment). Also you can use the specifier ALL,

Reviewer: ```suggestion
# The first part is the faction name, which is the name which is internally used by the game (see the /alignment command, or use the list on the wiki: https://lotrminecraftmod.fandom.com/wiki/Alignment#List_of_all_Alignment_Factions). You can also use the specifier ALL,
```

# which means that every player, regardless of his alignment, won't be able to use that item. Note that if you have blocked the usage of an item with ALL, you must make sure

> # that no other condition is there which would allow the usage of the item - otherwise the item is usable under that condition.

Reviewer: ```suggestion
# that there is no other condition which would allow the usage of the item - otherwise the item is usable under that condition.
```

#
# The second part is the minimum required alignment (see above). If the specified faction is ALL, the alignment value will be ignored (but has to be a valid number).

> # Additionally one can directly append :P to the alignment value, in that case, the player also has to pledge to the specified faction. (Example: 100:P would be the alignment value 100

Reviewer: Do not forget changing this if it is changed to :PLEDGE

Author: Done with the rebase, for the diff, see my comment below

> # Additionally one can directly append :P to the alignment value, in that case, the player also has to pledge to the specified faction. (Example: 100:P would be the alignment value 100

Reviewer: ```suggestion
# Additionally one can directly append :P to the alignment value, in that case, the player also has to pledge to the specified faction. (Example: 100:P would require at least 100 alignment
```

> # with also pledging required)

Reviewer: ```suggestion
# and pledging)
```

#
# The third part is optional - it's the condition, under which the entry will be applied. Valid values are PVP, PVE and ALWAYS. If not specified, ALWAYS is assumed.

> # Under this condition the items will be blocked, so, for example, with PVP as condition the specified items will be only blocked for PvP if the alignment criterion is not fulfilled.

Reviewer: ```suggestion
# Under this condition the items will be blocked. For example, with PVP as condition the specified items will only be blocked for PvP if the alignment criterion is not fulfilled.
```

#

> # The fourth part is a comma-separated list of unlocalized item names (e.g. lotr:item.coin), where the specified condition applies.

Reviewer: ```suggestion
# The fourth (or third if no condition is specified) part is a comma-separated list of unlocalized item names (for example, lotr:item.coin), where the specified condition applies.
```

#
# If this file is deleted, it'll be recreated with default configuration data.

> # On startup you'll see in the logs if the PvP Mode Mod found errors in this file - then you can fix them.

Reviewer: ```suggestion
# On startup you'll see in the logs if the PvP Mode Mod found errors in this file, so that you can fix them.
```

#
# Some words to the file format: 
# Comments cover a whole line, which must start with a #. The PvP Mode Mod will ignore them.
# Blank lines and spaces will be ignored too - you can put as much spaces between the configuration entries as you want.
#

# General

  # Generally blocked in PvP
  
  	# Mithril Armor, Sword, Daggers, Battleaxe, Warhammer, Spear, Halberd, Crossbow

>   	#ALL;0;PVP;lotr:item.helmetMithril,lotr:item.bodyMithril,lotr:item.legsMithril,lotr:item.bootsMithril,lotr:item.swordMithril,lotr:item.daggerMithril,lotr:item.daggerMithrilPoisoned,lotr:item.battleaxeMithril,lotr:item.hammerMithril,lotr:item.spearMithril,lotr:item.halberdMithril,lotr:item.mithrilCrossbow

Reviewer: Why is this condition commented out, but the others not?

Author: This is just a template for those who want to block that in PvP. As a default value we don't want that, but I heard of some servers that they would like it, and this line saves them some time.

Reviewer: Then why are the other lines not commented out? Some servers may not want those too.

Author: This file contains the default entries which we developed, so something must be enabled there. What most servers are going to change there, are probably the alignment values. It's also less work to remove unwanted lines than having to remove the `#` before many lines because one would like to use this (well, with Regex it's easy, but not everyone knows that).

Reviewer: Fair enough.
